lib/engine/game/g_1822_mx/game.rb

```diff
@@ -544,7 +544,8 @@ def operating_order
         end
 
         def active_players
-          return [@ndem_acting_player] if @ndem_acting_player
+          ndem_acting_player = @round.active_step.respond_to?(:ndem_acting_player) ? @round.active_step.ndem_acting_player : nil
+          return [ndem_acting_player] if ndem_acting_player
```

Comment:
return [@round.active_step.ndem_acting_player] if @round.active_step.respond_to?(:ndem_acting_player)

Reply:
Also, should you check if current_entity == ndem?

Reply:
Yep, changes incoming